_layouts/open-data-solutions.html

```diff
@@ -0,0 +1,1632 @@
+                                  
+                                          <li class=" w-full " role="presentation">
+                                            <a id="dashboard-styled-tab" data-tabs-target="#open-aerial-map" type="button" role="tab" aria-controls="dashboard" aria-selected="false" class="inline-flex items-center px-4 py-3 rounded-lg hover:text-gray-900 bg-gray-50 hover:bg-gray-100 w-full dark:bg-gray-800 dark:hover:bg-gray-700 dark:hover:text-white">
+                                                <svg width="10%"  viewBox="0 0 80 48" style="padding-right: 2%;"><g fill="#D73F3F" fill-rule="evenodd"><path d="M37.02656,20.77496 C37.02656,20.29096 37.12976,19.85896 37.33616,19.48616 C37.54096,19.11176 37.81296,18.80696 38.15056,18.56376 C38.48656,18.32136 38.86496,18.13736 39.28576,18.01336 C39.70496,17.89016 40.12736,17.82776 40.55376,17.82776 C40.97856,17.82776 41.40256,17.89016 41.82336,18.01336 C42.24176,18.13736 42.62016,18.32136 42.95616,18.56376 C43.29456,18.80696 43.56576,19.11176 43.77216,19.48616 C43.97856,19.85896 44.08096,20.29096 44.08096,20.77496 L44.08096,27.14696 C44.08096,27.64136 43.97856,28.07496 43.77216,28.44216 C43.56576,28.81096 43.29456,29.11416 42.95616,29.35736 C42.62016,29.60136 42.24176,29.78616 41.82336,29.90856 C41.40256,30.03176 40.97856,30.09336 40.55376,30.09336 C40.12736,30.09336 39.70496,30.03176 39.28576,29.90856 C38.86496,29.78616 38.48656,29.60136 38.15056,29.35736 C37.81296,29.11416 37.54096,28.81096 37.33616,28.44216 C37.12976,28.07496 37.02656,27.64136 37.02656,27.14696 L37.02656,20.77496 Z M39.05136,27.15016 C39.05136,27.56536 39.28256,27.87496 39.56576,28.07176 C39.84736,28.27016 40.18096,28.37496 40.56576,28.37496 C40.94896,28.37496 41.28416,28.27016 41.56656,28.07176 C41.84736,27.87496 42.00816,27.56536 42.00816,27.15016 L42.00816,20.77816 C42.00816,20.36136 41.84736,20.05256 41.56656,19.85576 C41.28416,19.65816 40.94896,19.52536 40.56576,19.52536 C40.18096,19.52536 39.84736,19.65816 39.56576,19.85576 C39.28256,20.05256 39.05136,20.36136 39.05136,20.77816 L39.05136,27.15016 Z"/><polygon points="33.379 17.815 33.379 22.99 30.152 22.99 30.152 17.815 28.387 17.815 28.387 30.1 30.152 30.1 30.152 24.923 33.379 24.923 33.379 30.1 35.278 30.1 35.278 17.815"/><polygon points="52.431 17.815 45.426 17.815 45.426 19.622 48.015 19.622 47.985 30.1 49.847 30.1 49.847 19.622 52.431 19.622"/><path d="M65.42344,28.7668 L79.67944,28.7668 L71.15544,19.1684 L62.66904,9.57 L54.18104,0.000400000001 L11.53944,0.000400000001 L20.02584,9.57 L5.73784,9.57 L14.22584,19.1684 L14.29144,19.2332 L0.000239999999,19.2332 L8.52184,28.8316 L17.01304,38.43 L25.50024,48.0004 L68.14184,48.0004 L59.65064,38.43 L59.58824,38.3364 L73.87864,38.3364 L65.42344,28.7668 Z M39.83944,39.9492 C30.31864,39.9492 22.60104,32.8084 22.60104,24.0012 C22.60104,15.1908 30.31864,8.05 39.83944,8.05 C49.36104,8.05 57.07944,15.1908 57.07944,24.0012 C57.07944,32.8084 49.36104,39.9492 39.83944,39.9492 Z"/></g></svg>
```

Comment:
svg I suppose ?

Reply:
yes, idk why for these I chose svg, should I go for <i> instead?